app/models/activity.rb

```diff
@@ -1,9 +1,11 @@
 class Activity < ApplicationRecord
-  belongs_to :user
+  self.ignored_columns = [:user_id]
```

Comment:
There's a gap between migrations running and processors coming online; so by adding the ignored column I can do a follow up migration to drop the column.